Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1184,6 +1188,18 @@ def convert_all_unix_keys_to_date(incident: dict) -> dict:
     return incident
 
 
+def convert_all_unix_keys_to_date_user(incident: dict) -> dict:
+    keys = ['firstSeen', 'lastSeen', 'lastActivityTime', 'endTime', 'startTime']
+    if 'user' in incident:
+        for key in keys:
+            if key in incident['user']:
+                incident['user'][key] = convert_unix_to_date(incident['user'][key]).split('.')[0] + 'Z'
+            if key in incident['highestRiskSession'] and incident['highestRiskSession'][key]:
```

Comment:
incident['highestRiskSession'][key] can be exist but None? if not remove the and condition